apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -122,30 +110,28 @@ function CtaSection(props: CtaProps) {
   return (
     <Box className="support-cta-container" sx={{ borderTopColor: 'grey.200' }}>
       <div style={{ flex: 4, height: '100%' }}>{ctaBlurb}</div>
+
       <PrimaryButton
+        href="https://wa.me/6583668579"
+        target="_blank"
+        rel="noopener"
         sx={{ flex: 5, backgroundColor: 'primary.700', height: '100%' }}
       >
+        <WhatsappIcon></WhatsappIcon>
```

Comment:
@e056 this and the entire primary button is the only thing that's new right? Can we make it into a component that takes in props instead?

The props will be for:
- The link to redirect to
- The icon to be used (not all will be WA)
- The text for the button label